apigee_sync_test.go

```diff
@@ -5,90 +5,283 @@ import (
 	"github.com/apigee-labs/transicator/common"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"net/http/httptest"
+	//"time"
 )
 
-var _ = Describe("listener", func() {
+var _ = Describe("Sync", func() {
 
-	It("should bootstrap from local DB if present", func(done Done) {
+	Context("Sync", func() {
 
-		expectedTables := make(map[string]bool)
-		expectedTables["kms.company"] = true
-		expectedTables["edgex.apid_cluster"] = true
-		expectedTables["edgex.data_scope"] = true
+		var initializeContext = func() {
```

Comment:
Is it necessary to make these explicit functions instead of using beforeEach() and afterEach()?